src/main/java/org/trigon/hardware/misc/leds/LEDStrip.java

```diff
@@ -0,0 +1,140 @@
+package org.trigon.hardware.misc.leds;
+
+import com.ctre.phoenix.led.LarsonAnimation;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.trigon.hardware.RobotHardwareStats;
+
+import java.util.function.Supplier;
+
+/**
+ * A wrapper class for LED strips. This class provides a set of methods for controlling LED strips.
+ */
+public abstract class LEDStrip extends SubsystemBase {
+    public static LEDStrip[] LED_STRIPS = new LEDStrip[0];
```

Comment:
adding this comment so that this doesn't get accidently merged, I'm still working on a but with the LED_STRIPS array.

Reply:
what's the bug?

Reply:
where is it used exactly?

Reply:
in the setDefaultCommandForAllLEDs method, and as a parameter to be used in the commands.